pyproject.toml

```diff
@@ -115,7 +115,7 @@ omit = [
     "*/examples/*",
     "*/setup.py",
     "*/tasks.py",
-    "*/src/pykiso/tool/*",
+    "*/src/pykiso/tool/testrail/*",
```

Comment:
Why only exclude testrail tool just before we had all package in tool folder?

Reply:
For the testrail it was just because some files like the cli.py and console.py for testrail had no test so I was not sure if I should just skip these files or all the repository for the code coverage.